packages/clerk-js/src/ui/primitives/Button.tsx

```diff
@@ -31,10 +31,13 @@ const { applyVariants, filterProps } = createVariants((theme, props: OwnProps) =
       textVariant: common.textVariants(theme),
       size: {
         iconLg: { minHeight: theme.sizes.$13, width: theme.sizes.$13 },
-        xs: { minHeight: theme.sizes.$1x5, padding: `${theme.space.$1} ${theme.space.$2}` },
+        xs: {
+          minHeight: theme.sizes.$7x5,
+          padding: `${theme.space.$1} ${theme.space.$3}`,
+        },
         sm: {
-          minHeight: theme.sizes.$7,
-          padding: `${theme.space.$1x5} ${theme.space.$3x5}`,
+          minHeight: theme.sizes.$8x5,
+          padding: `${theme.space.$1x5} ${theme.space.$3}`,
```

Comment:
I'm not sure about this change. It may has impact on other components that use these button sizes.

Reply:
`xs` is new, so only the "Upload" button on the create org component uses it.
As for `sm`, if it fixes the size for this component, wouldn't we want that also fixed in the other places?

Reply:
Forgot to mention this was a request from the design team.

<img width="430" alt="Screenshot 2024-01-16 at 10 28 59" src="https://github.com/clerk/javascript/assets/12413903/5f2de33e-bc72-441e-96e3-2f4f5dd5caf6">

Reply:
The values are correct here!